### PR TITLE
[WIP] Update project to start with cordova prepare

### PR DESCRIPTION
**Pull request by kosssi**

### CODE REVIEW

File: src/app/lib/file_cache_handler.coffee [hunk 1]
```diff
@@ -214,3 +216,16 @@ module.exports = class FileCacheHandler
                 fs.rmrf binaryFolder, (err) ->
                     return callback err if err and err.code is not 1
                     callback()
+
+
+    open: (url) ->
+        success = (entry) =>
+            entry.file (file) =>
```
Comment (nono) [inline src/app/lib/file_cache_handler.coffee]: I think the fat arrows are not necessary here

File: src/app/views/folder_line.coffee [hunk 2]
```diff
@@ -86,20 +86,19 @@ module.exports = class FolderLineView extends BaseView
...
-                (success) -> , # do nothing
-                (err) ->
-                    if 0 is err?.indexOf 'No Activity found'
-                        err = t 'no activity found'
-                    log.error err
-                    alert err.message
+        cozyFile = @model.attributes
+        if @fileCacheHandler.isSameBinary cozyFile
+            @fileCacheHandler.getBinaryUrl cozyFile, (err, url) =>
+                @fileCacheHandler.open url
```
Comment (nono) [inline src/app/views/folder_line.coffee]: You don't check `err`. If it is volontary, you should add a comment to explain why

**Comment by frankrousseau** [comment]
@kosssi Don't forget to make Travis green ;).

**Comment by kosssi** [comment]
I close this PR because I separate each commit for specific branch (android or ios)

**Pull request closed without merging**